test-data/cr_nemo_update.yml

```diff
@@ -1,11 +0,0 @@
----
-
-apiVersion: stable.nicolerenee.io/v1
-kind: Character
-metadata:
-  name: nemo
-spec:
-  Name: Nemo
-  From: Finding Nemo
-  By: Disney
-  For: Reasons
```

Comment:
This is removed on make clean, so it should probably not be checked in to the codebase.

Reply:
This file is required. It should not be removed by `make clean`.